app/src/main/java/com/example/sunmusic/screen/trending/TrendingFragment.kt

```diff
@@ -4,14 +4,58 @@ import android.os.Bundle
 import android.view.View
 import androidx.fragment.app.Fragment
 import com.example.sunmusic.R
+import com.example.sunmusic.data.model.Album
+import com.example.sunmusic.data.model.Track
+import com.example.sunmusic.data.repository.AlbumRepositoryImpl
+import com.example.sunmusic.data.repository.TrackRepositoryImpl
+import com.example.sunmusic.utils.toast
+import kotlinx.android.synthetic.main.fragment_trending.view.*
 
-class TrendingFragment : Fragment(R.layout.fragment_trending) {
+class TrendingFragment : Fragment(R.layout.fragment_trending), TrendingContract.View {
+    private lateinit var adapter: TrendingAdapter
+    private lateinit var presenter: TrendingPresenter
```

Comment:
https://viblo.asia/p/co-nen-su-dung-kotlin-lateinit-var-hay-khong-LzD5dbneZjY
```suggestion
    private val adapter by lazy {
        TrendingAdapter()
    }
    private var presenter: TrendingPresenter? = null
```